pressbooks/pressbooks-network-catalog
# fix: line clamp on Firefox/Safari

Issue #46 

This PR aims to fix an issue with the line clamp feature on Firefox and Safari. It should not break the way the content (paragraphs, lists, links, etc.) is displayed.

### How to test

1. Add long descriptions to books (use multiple paragraphs, lists, etc.)
2. Open the catalog page and make sure the content is rendered properly without overlapping

>**Note**
Test against Chrome, Firefox, and Safari if possible
